src/pages/Bookmark/Bookmark.js

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import PostCard from "../../components/Posts/PostCard";
+
+const Bookmark = () => {
+  const { posts, bookmarks } = useSelector((store) => store.posts);
+  const { user, token } = useSelector((state) => state.auth);
+  const [bookmark, setBookmarks] = useState([]);
+
+  useEffect(() => {
+    try {
+      const getBookmarks = async () => {
+        const response = await axios.get("/api/users/bookmark", {
+          headers: { authorization: token },
+        });
+        setBookmarks(response.data.bookmarks);
+      };
+      getBookmarks();
+    } catch (error) {
+      console.log(error);
+    }
+  }, [posts, token, user, bookmarks]);
+
+  const bookmarkedPosts = posts.filter((post) =>
+    bookmark.some((bookmark) => bookmark === post._id)
+  );
+
+  return (
+    <div>
```

Comment:
Add title at the top of page saying bookmarks , so user will know that he/she is on bookmarks page